lib/ansible/cli/config.py

```diff
@@ -262,11 +266,17 @@ def _list_entries_from_args(self):
         '''
         build a dict with the list requested configs
         '''
+
         config_entries = {}
         if context.CLIARGS['type'] in ('base', 'all'):
             # this dumps main/common configs
             config_entries = self.config.get_configuration_definitions(ignore_private=True)
 
+            # for base and all, we include galaxy servers
+            config_entries['GALAXY_SERVERS'] = {}
+            for server in self._galaxy_servers:
+                config_entries['GALAXY_SERVERS'][server.upper()] = self.config.get_configuration_definitions('galaxy_server', server)
```

Comment:
perhaps make it a dict comprehension so that it's a single instruction?
```suggestion
            config_entries['GALAXY_SERVERS'] = {
                server.upper(): self.config.get_configuration_definitions('galaxy_server', server)
                for server in self._galaxy_servers
            }
```